example/src/main.rs

```diff
@@ -67,7 +69,8 @@ fn sync_example() {
     // clients, as well as the total number of expected clients (or None if server can accept any
     // number of clients).  A thread will be spun up to wait for extra clients beyond the number
     // to block on.
-    let server_config = ServerConfig::new(server_addr, 0, Some(1), 3, 2);
+    let mut server_config = ServerConfig::new(server_addr, 0, Some(1), 3, 2);
+    server_config.with_alpn_protocols(alpn_protocols);
```

Comment:
I would like to keep the actual example code a minimal example.  Could you either add this as a comment like:

```
// Here you can set various optional configurations
// server_config.with_alpn_protocols(&[b"hq-29"]);
```

Same with the client config below

Reply:
Definitely.